contracts/SportMarkets/Parlay/ParlayMarket.sol

```diff
@@ -228,120 +141,31 @@ contract ParlayMarket is OwnedWithInit {
 
     function exerciseWiningSportMarkets() external onlyAMM {
         require(!paused, "Market paused");
-        require(
-            numOfAlreadyExercisedSportMarkets < numOfSportMarkets && numOfResolvedSportMarkets < numOfSportMarkets,
-            "Already exercised all markets"
-        );
-        for (uint i = 0; i < numOfSportMarkets; i++) {
-            _updateSportMarketParameters(sportMarket[i].sportAddress, i);
-            if (sportMarket[i].resolved && !sportMarket[i].exercised) {
-                _exerciseSpecificSportMarket(sportMarket[i].sportAddress, i);
-            }
-        }
-        if (parlayAlreadyLost && !fundsIssued) {
-            uint totalSUSDamount = parlayMarketsAMM.sUSD().balanceOf(address(this));
+        require(isParlayExercisable(), "Parlay not exercisable yet");
+        require(areAllPositionsResolved() || isParlayLost(), "Parlay already exercised");
```

Comment:
it is bit confusing since:
`isParlayExercisable() ? !resolved && areAllPositionsResolved() || isParlayLost()`

then again is the same OR check, 
I guess line 144 can be changed or removed

Reply:
I think like 145 can be removed